README.md

```diff
@@ -3,128 +3,6 @@
 This Earthdata Cloud Cookbook is being developed by the [NASA-Openscapes team](https://nasa-openscapes.github.io/).
 
 ### How to contribute to our book
-Information for our team [to contribute to the Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/contributing/).
+Information for our team [to contribute to the Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/contributing/). This includes setup and workflow instructions.
```

Comment:
Nice, I think this is a good approach to not overwhelm users when they open the repo.